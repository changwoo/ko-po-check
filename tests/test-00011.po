# -*-text-*-
# UI
#
msgid ""
msgstr ""
"Project-Id-Version: test 9\n"
"POT-Creation-Date: 2002-08-01 01:30+0900\n"
"PO-Revision-Date: 2002-08-01 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.c:1
msgid "Pre_ferences..."
msgstr "설정(_F)..."

# "..." 빠짐
#: test.c:2
msgid "Open..."
msgstr "열기"

# 접근키 틀림
#: test.c:2
msgid "S_earch"
msgstr "찾기(_S)"
