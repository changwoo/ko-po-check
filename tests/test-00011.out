test-00011.po:17: 설정(_F)...: 다음과 같이 번역해야 합니다: "기본 설정(_F)..."
test-00011.po:22: 번역문이 원문과 같이 ...으로 끝나야 합니다
test-00011.po:22: 열기: 다음과 같이 번역해야 합니다: "열기..."
test-00011.po:27: 'E' vs 'S': 원문과 번역문의 mnemonic이 다릅니다
test-00011.po:27: 찾기(_S): 다음과 같이 번역해야 합니다: "찾기(_E)"
